File: resolucoes/java/lista01/src/main/java/org/example/Ex004.java
/*
 * Desenvolva um algoritmo que leia dois números inteiros e mostre o somatório entre eles. ex:
 * Digite um valor: 8
Digite outro valor: 5
A soma entre 8 e 5 é igual a 13.
 */

package org.example;

import java.util.Scanner;

public class Ex004 {
    public static void main(String[] args) {
        Ex004.main();
    }

    public static void main() {

        Scanner sc = new Scanner(System.in);
        System.out.print("Digite um valor: ");
        int n1 = sc.nextInt();
        System.out.print("Digite outro valor: ");
        int n2 = sc.nextInt();
        System.out.printf("A soma entre %s e %s é %s", n1, n2, n1 + n2);
        sc.close();

    }

}

File: resolucoes/php/lista01/ex004.php
<?php
/*Desenvolva um algoritmo que leia dois números inteiros e mostre o somatório entre eles. ex:

Digite um valor: 8
Digite outro valor: 5
A soma entre 8 e 5 é igual a 13.
*/

$valor1 = (float)readline('Digite um valor A: ');
$valor2 = (float)readline('Digite um valor B: ');

$total = $valor1 + $valor2;

echo "\rA soma entre $valor1 e $valor2 é igual a $total";